scripts/diff.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# This script extracts files changed between two specified Git commit IDs and performs various file operations.
+# Usage:
+# 1. Make sure this script is executable: `chmod +x ./scripts/diff.sh`
+# 2. Run the script: `./scripts/diff.sh`
+# 3. The script will prompt you to enter two commit IDs.
+#    - `commit_id_from` (inclusive): The starting commit ID for the diff operation.
+#    - `commit_id_to`: The ending commit ID for the diff operation.
+# 4. The script will create a tarball of the changed files, extract them, and organize them into specified directories.
+# 5. It will then move PDF and SVG files to separate directories and open these directories.
+# 6. Finally, it will list files with more than 3 lines and perform a check in the all_files_svg directory.
+
+# Note:
+# - Commit IDs can be obtained using `git log` to view the commit history.
+# - The script assumes it is located in a `scripts` directory relative to the Git repository.
+
+# Ensure the script is run from the correct directory
+# Do not delete this check, this scripts needs to be ran at the root of the repo.
+if [[ $(basename "$PWD") == "scripts" ]]; then
+    echo "Please run this script from the root directory of your repository."
+    exit 1
+fi
+
+# Prompt the user for the commit IDs
+read -p "Enter the commit ID from (inclusive): " commit_id_from
+read -p "Enter the commit ID to: " commit_id_to
```

Comment:
Is it possible to just run this with a from and no "to" so it runs to the current commit?